### ISSUE TITLE
Linux: SETFLUIDSdll err 101: error opening file

### ISSUE THREAD

**Issue by hugo-cwp** [usnistgov/REFPROP-cmake#82]
Dear Ian,

We bought and have been using NIST REFPROP 10.0 in Windows without issues. We use ctREFPROP from https://github.com/usnistgov/REFPROP-wrappers to make use of the library from python.
We have just compiled a .so to hopefully  run our python scripts in a Linux system We  followed your instructions for compilation, and started testing the code:

We seem to be able to construct the REFPROPInstance using the .so file just fine like this:

```python
extension=get_default_DLL_extension() # unnecessary, just to check that .so file is indeed used. It is the case
RP = REFPROPFunctionLibrary(RP_PATH,extension) 
```

So far so good. 
However, when trying to make use of a fluid:
```python
def check_fluid(fluid:str):
    global RP
    ierr=RP.SETFLUIDSdll(fluid.upper())
    if ierr != 0:
        # throw error
        raise ValueError(f'❌ Fluid {fluid} not found in REFPROP FLD files')
    else:
        return True
```
...our calls to ``SETFLUIDSdll`` return error 101 (error opening file), as if the FLD file was not found in the FLUIDS folder (it is actually there). This works on Windows but not on Linux.
        
Question: Are we using the correct method? If so, should we somehow specify the path to the FLUIDS folder specifically in Linux?

Thank you for your help. Happy to provide more information if required.

**Comment by ianhbell** [comment]
Make sure you call ``SETPATHdll`` right after making the RP instance so that REFPROP knows where to look for fluid files. Also, global variables are not a great idea, better to pass to the function.

**Comment by ianhbell** [comment]
Note: I tidied up your issue repot

**Comment by hugo-cwp** [comment]
Thank you. Still no luck but the result is different:

Edited:

```python
RP_PATH = os.path.join(os.path.dirname(__file__), '..','resources','REFPROP')
extension=get_default_DLL_extension()
print(f'RP_PATH: {RP_PATH}')
print(f'extension: {extension}')
RP = REFPROPFunctionLibrary(RP_PATH,extension)
RP.SETPATHdll(RP_PATH)
```


```python
def check_fluid(RP,fluid:str):
    ierr=RP.SETFLUIDSdll(fluid.upper())
    if ierr != 0:
        # throw error
        raise ValueError(f'❌ Fluid {fluid} not found in REFPROP FLD files. Error code: {ierr}')
    else:
        return True
```

The value for ierr changed from 101 to **-101**. Not sure what to make of it, could not find it in the [documentation](https://www.nist.gov/system/files/documents/2018/05/23/refprop10a.pdf)

**Comment by ianhbell** [comment]
Check the herr

**Comment by ianhbell** [comment]
Sorry, here it is ``RP.ERRMSG(ierr)`` (only for ``SETFLUIDSdll``)

**Comment by hugo-cwp** [comment]
```
Error in opening fluid file for component 21, which may be required for the calculation of transport properties: filename = NITROGEN.FLD
```

...which is absolutely right, it's missing, as well as some others. I placed them under FLUIDS and the issue was solved.

So it turns out I've been using the dll and FLD files from the default windows setup location all along (I only realized now that I'm attempting to run in a different system).

Thank you very much for your super fast response.

Regards,
Hugo
https://www.cwp.global/hydrogen/
